github_connector/__openerp__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# @author: Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Github Connector',
+    'summary': 'Recover information from github repositories',
```

Comment:
Synchronize information from Github repositories
(@legalsylvain I cannot push to your repo because of the ACL)